modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/BoundObjectDefinitionsExportImportTest.java

```diff
@@ -0,0 +1,259 @@
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.vulcan.pagination.Page;
+import com.liferay.portal.vulcan.pagination.Pagination;
+
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Guilherme Sá
+ */
+@RunWith(Arquillian.class)
+public class BoundObjectDefinitionsExportImportTest
+	extends BaseExportImportTestCase {
+
+	@ClassRule
+	@Rule
+	public static final LiferayIntegrationTestRule liferayIntegrationTestRule =
+		new LiferayIntegrationTestRule();
+
+	@Before
+	public void setUp() throws Exception {
+		user = TestPropsValues.getUser();
+
+		ObjectDefinitionResource.Builder builder =
+			_objectDefinitionResourceFactory.create();
+
+		_objectDefinitionResource = builder.user(
+			user
+		).build();
+
+		Class<?> clazz = getClazz();
+
+		_defaultObjectDefinitionJSONString = StringUtil.read(
+			clazz.getResourceAsStream(
+				"dependencies/test-default-object-definition.json"));
+
+		_defaultObjectRelatioshipJSONString = StringUtil.read(
+			clazz.getResourceAsStream(
+				"dependencies/test-default-object-relationship.json"));
+	}
+
+	@Test
+	public void testExportImportBoundObjectDefinitions() throws Exception {
+
+		// Import bound object definitions as draft
+
+		String draftObjectDefinitionsJSONString = String.valueOf(
+			JSONUtil.putAll(
+				_jsonFactory.createJSONObject(
+					_defaultObjectDefinitionJSONString
+				).put(
+					"externalReferenceCode", "TESTOBJECTDEFINITION1"
+				).put(
+					"name", "TestObjectDefinition1"
+				).put(
+					"objectRelationships",
+					JSONUtil.put(
+						_createOneToManyObjectRelationship(
+							"TESTOBJECTDEFINITION1", "TESTOBJECTDEFINITION2",
+							"TestObjectDefinition2", "objectRelationship1"))
+				).put(
+					"rootObjectDefinitionExternalReferenceCode",
+					"TESTOBJECTDEFINITION1"
+				),
+				_jsonFactory.createJSONObject(
+					_defaultObjectDefinitionJSONString
+				).put(
+					"externalReferenceCode", "TESTOBJECTDEFINITION2"
+				).put(
+					"name", "TestObjectDefinition2"
+				).put(
+					"objectRelationships",
+					JSONUtil.put(
+						_createOneToManyObjectRelationship(
+							"TESTOBJECTDEFINITION2", "TESTOBJECTDEFINITION3",
+							"TestObjectDefinition3", "objectRelationship2"))
+				).put(
+					"rootObjectDefinitionExternalReferenceCode",
+					"TESTOBJECTDEFINITION1"
+				),
+				_jsonFactory.createJSONObject(
+					_defaultObjectDefinitionJSONString
+				).put(
+					"externalReferenceCode", "TESTOBJECTDEFINITION3"
+				).put(
+					"name", "TestObjectDefinition3"
+				).put(
+					"rootObjectDefinitionExternalReferenceCode",
+					"TESTOBJECTDEFINITION1"
+				)));
+
+		testExportImportJSONString(
+			draftObjectDefinitionsJSONString, draftObjectDefinitionsJSONString,
+			null, "TestObjectDefinition1");
+
+		// Cannot publish hierarchy via import
```

Comment:
I would not use the word `hierarchy`. 
I would use: Cannot import bound object definitions as published